aks/extra-variables.tf

```diff
@@ -5,5 +5,4 @@ variable "logs_storage" {
     storage_account     = string
     storage_account_key = string
```

Comment:
Shouldn't this argument `storage_account_key` be removed if it's to be passed in a separate variable?

Reply:
It doesn't necessarily have to be passed in the `sensitive_values` variable. It will be marked sensitive though within the module. The same logic applies to the other attributes of the `logs_storage` object. For me, it's still the same unanswered question of what to pass as a separate variable and what to passe as "raw" helm values.